integration-tests/actions/vrf/vrfv2plus/contract_steps.go

```diff
@@ -652,3 +652,58 @@ func CancelSubsAndReturnFunds(ctx context.Context, vrfContracts *vrfcommon.VRFCo
 		}
 	}
 }
+
+func FundWrapperConsumer(
+	sethClient *seth.Client,
+	subFundingType string,
+	linkToken contracts.LinkToken,
+	wrapperConsumer contracts.VRFv2PlusWrapperLoadTestConsumer,
+	vrfv2PlusConfig *vrfv2plusconfig.General,
+	l zerolog.Logger,
+) error {
+	fundConsumerWithLink := func() error {
+		//fund consumer with Link
+		linkAmount := big.NewInt(0).Mul(big.NewInt(1e18), big.NewInt(*vrfv2PlusConfig.WrapperConsumerFundingAmountLink))
+		l.Info().
+			Str("Link Amount", linkAmount.String()).
+			Str("WrapperConsumerAddress", wrapperConsumer.Address()).Msg("Funding WrapperConsumer with Link")
+		return linkToken.Transfer(
+			wrapperConsumer.Address(),
+			linkAmount,
+		)
+	}
+	fundConsumerWithNative := func() error {
+		//fund consumer with Eth (native token)
+		_, err := actions.SendFunds(l, sethClient, actions.FundsToSendPayload{
+			ToAddress:  common.HexToAddress(wrapperConsumer.Address()),
+			Amount:     conversions.EtherToWei(big.NewFloat(*vrfv2PlusConfig.WrapperConsumerFundingAmountNativeToken)),
+			PrivateKey: sethClient.PrivateKeys[0],
+		})
+		return err
+	}
+	var err error
```

Comment:
nit: if you are going to do early return in the case statements, then I think having a locally scoped variable would be better.

```go
switch vrfv2plusconfig.BillingType(subFundingType) {
case vrfv2plusconfig.BillingType_Link:
	err := fundConsumerWithLink()
	if err != nil {
		return err
	}
case vrfv2plusconfig.BillingType_Native:
	err := fundConsumerWithNative()
	if err != nil {
		return err
	}
case vrfv2plusconfig.BillingType_Link_and_Native:
	err := fundConsumerWithLink()
	if err != nil {
		return err
	}
	err = fundConsumerWithNative()
default:
	return fmt.Errorf("invalid billing type: %s", subFundingType)
}
return nil
```